src/main/java/org/aion/wallet/dto/AccountDTO.java

```diff
@@ -11,6 +12,7 @@
     private String balance;  //TODO this has to be BigInteger
     private String name;
     private boolean active;
+    private BufferedImage qrCode;
```

Comment:
The QR code does not change for an account so it should be final, the same as the `publicAddress` and it won't need a setter.
Considering this, it should be initialized in the constructor which removes some duplicated calls from the AccountManager:
`QRCodeUtils.writeQRCode(account.getPublicAddress(), 150, 150)`

Also, `150` can be a constant in the AccountDTO or the QRCodeUtils class

Reply:
if we want to make it dynamic, this might not be constant anymore.
this is the reason of keeping 150, 150 until this moment. should I change it to constant?